pkg/evpn/interface.go

```diff
@@ -83,7 +83,7 @@ func (s *Server) CreateInterface(_ context.Context, in *pb.CreateInterfaceReques
 			return nil, err
 		}
 		// now get Subnet/Bridge to plug this port/interface into
-		bridge, ok := s.Subnets[ifaceobj.VpcNameRef]
+		bridge, ok := s.Bridges[ifaceobj.VpcNameRef]
```

Comment:
VPCNameRef?

Reply:
ignore for now, will be refactored part of SVI and PORT changes